stix_shifter_modules/aws_guardduty/configuration/config.json

```diff
@@ -0,0 +1,35 @@
+{
+    "connection": {
+        "type": {
+            "displayName": "AWS GuardDuty",
+            "group": "aws"
+        },
+        "region": {
+            "type": "text",
+            "previous": "connection.options.region"
```

Comment:
remove ` "previous": "connection.options.region"`

Reply:
Removed this attribute and the changes are present in Actual AWS GuardDuty PR #1525